### PR TITLE
Modified volume deleted flag check

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Tendrl/monitoring-integration/issues/375

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n please review this

File: tendrl/monitoring_integration/sync/gluster_cluster_details.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def get_volumes_details(cluster_key):
             deleted = etcd_utils.read(
                 cluster_key + "/Volumes/" + str(volume_id) + "/" + "deleted"
             ).value
-            if str(deleted).lower() == "false":
+            if str(deleted).lower() != "true":
```
Comment (GowthamShanmugam) [inline tendrl/monitoring_integration/sync/gluster_cluster_details.py]: Volumes delete flag default value is None only, So we can't check flag is false

Comment (shtripat) [reply tendrl/monitoring_integration/sync/gluster_cluster_details.py]: In case default None str(deleted) would work?

Comment (shtripat) [reply tendrl/monitoring_integration/sync/gluster_cluster_details.py]: In case default None str(deleted) would work?

Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/sync/gluster_cluster_details.py]: yes it will work, i tested this

Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/sync/gluster_cluster_details.py]: str(None) is 'None' only, it won't raise any error

Comment (shtripat) [reply tendrl/monitoring_integration/sync/gluster_cluster_details.py]: ok

**Pull request merged**